generators/base-docker/docker-base.mjs

```diff
@@ -99,9 +99,14 @@ export function loadConfigs() {
   this.logger.debug(`Apps folders: ${this.appsFolders}`);
   this.appsFolders.forEach((appFolder, index) => {
     const path = this.destinationPath(`${this.directoryPath + appFolder}`);
+    this.logger.debug(chalk.red.bold(`App folder ${path}`));
     if (this.fs.exists(`${path}/.yo-rc.json`)) {
       const config = getConfigWithDefaults(removeFieldsWithNullishValues(this.getJhipsterConfig(`${path}/.yo-rc.json`).getAll()));
       config.composePort = serverPort + index;
+      this.logger.debug(
+        chalk.red.bold(`${config.baseName} has compose port ${config.composePort} and appIndex ${config.applicationIndex}`)
+      );
```

Comment:
Are these log messages still necessary, or can they be removed?

Reply:
They are debug logs so I would keep them

Reply:
I saw the docker-compose subgenerator does not assign the same port to the gateway when re-generating with `--skip-prompts`. I was not sure if it is still a bug. I don't think it affects the kubernetes subgenerator.